### ISSUE TITLE
[studio] There's a typo for error code 12002 when pulling from remote

### ISSUE THREAD

**Issue by alhambrav** [craftercms/craftercms#3396]
## Describe the bug
There's a typo for error code 12002 when pulling from remote:

`Pull from remote repository resulted in conflict. Resolve conflicts before continue working with repositroy`

## To Reproduce
1. Create a site parent based on Editorial BP
2. Create a site child based on parent by creating using create from remote with a URL to remote (use a local path, like `/home/user/code/crafter/craftercms/crafter-authoring/data/repos/sites/parent/sandbox`
3. Edit the homepage on the child site
4. Edit the homepage on the parent site
5. In your child site, go to SiteConfig > Remote Repositories and do a pull.  For the merge strategy, pick `None`
6. Notice there's a typo in the error message

## Expected behavior
Fix typo in error code 12002 and maybe reword the sentence so it says something like:
`Pull from remote repository resulted in conflict. Resolve conflicts before continuing work with repository`

## Screenshots
![Screen Shot 2019-08-23 at 3 52 41 PM](https://user-images.githubusercontent.com/25483966/63619971-1fd4ab00-c5be-11e9-91a4-a34db44efd3e.png)

## Logs
{{If applicable, attach the logs/stack trace (use https://gist.github.com).}}

### Version
Studio Version Number: 3.1.1-SNAPSHOT-077fde
Build Number: 077fde129e0a1aeb1bd8fc97fc27b80db6345607
Build Date/Time: 08-23-2019 11:38:33 -0400

### OS
OS X

### Browser
Chrome browser

## Additional context
{{Add any other context about the problem here.}}

**Comment by dejan-brkic** [comment]
Done
PR:
https://github.com/craftercms/studio/pull/1719
